src/components/Textarea/styled.ts

```diff
@@ -0,0 +1,88 @@
+import { InputStatus } from '@components/Textarea/Textarea';
+import styled, { css } from 'styled-components';
+
+export const StyledTextarea = styled.textarea<{
+  disabled: boolean | undefined;
+  status?: InputStatus;
+  isScrollable: boolean;
+}>`
+  width: 100%;
+  max-height: 80px;
+  min-height: 40px;
+
+  padding: 6px 14px;
+
+  background: #ffffff;
+  border: 2px solid #ebebeb;
+  border-radius: 10px;
+
+  font-size: 20px;
+  line-height: 24px;
+
+  resize: none;
+
+  overflow: hidden;
+  outline: none;
+
+  color: #323232;
+
+  &::placeholder {
+    font-weight: 300;
+    color: #737373;
+  }
+
+  &:hover,
+  &:focus {
+    border-color: #7ec0ee;
+  }
+
+  ${({ disabled }) =>
+    disabled &&
+    css`
+      border-color: #f14747;
```

Comment:
When the textarea is **disabled**, the border is colored in the color of the error.